napari/layers/shapes/_shapes_models/shape.py

```diff
@@ -164,6 +165,11 @@ def dims_displayed(self):
         """tuple: Dimensions that are displayed."""
         return self.dims_order[-self.ndisplay :]
 
+    @property
+    def bounding_box(self) -> np.ndarray:
+        """np.ndarray: 9x2 array of vertices of the interaction box."""
+        return self._bounding_box[:, self.dims_displayed]
```

Comment:
the docstring doesn't seem to match the code?

Reply:
updated